packages/telemetry/README.md

```diff
@@ -0,0 +1,3 @@
+# Astro Telemetry
+
+TODO
```

Comment:
TODO! Simple docs here would be fine, even linking to the docs.astro.build page on telemetry